### DISCUSSION TITLE
Enabling organization discussions leads to wrong notification reference

### DISCUSSION THREAD

**Discussion by Kludex** [community/community#30524] [Discussions]
Hi 👋 

@encode just enabled discussions on the organization, and now notifications from projects on that organization have 404.

This notification is from [httpx](https://github.com/encode/httpx):
<img width="1416" alt="image" src="https://user-images.githubusercontent.com/7353520/186149744-70129fc5-2ad6-4738-8059-559d6716ea2f.png">

This is the one: https://github.com/encode/httpx/discussions/2345.

If I click on the notification:
<img width="1292" alt="image" src="https://user-images.githubusercontent.com/7353520/186149875-2146618b-962e-4955-8684-2b0e07954425.png">

It goes to https://github.com/orgs/encode/discussions/2345 instead of https://github.com/encode/httpx/discussions/2345.

**Comment by lovelydinosaur** [comment]
Shall we start by turning this off, and seeing if it resolves?

**Reply by Kludex** [reply]
Yes, please.
